docs/recipe_development.rst

```diff
@@ -0,0 +1,132 @@
+==============
+Recipe Development
+==============
+
+mBuild encourages users to subclass ``Compound`` to disseminate reproducible scripts used for building their systems.
+Sharing the script used to construct the initial configuration and apply force field parameters (if applicable) is a great step towards increasing the reproducibility of the system initialization step.
+
+However, there may be cases where those subclasses and/or scripts can be generalized to support a broad range of systems.
+Such subclasses would be a valuable resource for many researchers, and might justify development of a Python package that could be distributed to the community.
+
+
+mBuild has been developed with this in mind, in the form of a plug-in recipe system.
+Detailed below are the specifications of this system, how to convert your own Python project into an mBuild-discoverable plug-in, and an example.
+
+Entry Points
+--------
+
+The basis of the recipe system in mBuild relies on `setuptools.entry_points <https://packaging.python.org/guides/creating-and-discovering-plugins/#using-package-metadata>`_.
```

Comment:
Not sure about this wording, I suggest something like `The basis of the recipe system is` or `The recipe system relies on`